fontforge/splineutil.c

```diff
@@ -7100,13 +7100,13 @@ return( changed );
 		    (&ptspace[i]->prevcp.x)[is_y]-cur<within ) {
 		(&ptspace[i]->prevcp.x)[is_y] = cur;
 		if ( (&ptspace[i]->prevcp.x)[!is_y]==(&ptspace[i]->me.x)[!is_y] )
-		    ptspace[i]->noprevcp = true;
+		    ptspace[i]->prevcp = ptspace[i]->me;
```

Comment:
flag?

Somewhat unrelated to this change, but this function is also kind of broken in general, it returns a `changed` flag, but not all return paths specify a return value.

Reply:
It's not as bad as it looks because the infinite for loop has no breaks, so the only way "out" is via that one return (right?). So the problem is that `changed` isn't being initialized to false. That mostly means there are bugs where the layer isn't being preserved for an undo.

Reply:
Actually there's no problem here -- changed is a parameter.

Reply:
Ah yep, missed the loop surrounding the code, these mixed tabs/spaces are a real chore to work with (in vscode at least...)